tests/routes/test_route_objects.py

```diff
@@ -0,0 +1,89 @@
+import geopandas as gpd
+import pytest
+
+import routes.route as route
+
+
+def test_create_route_object(monkeypatch: pytest.MonkeyPatch) -> None:
+    def _get_num_people_on_route(
+        route_path: list[str], population_data: gpd.GeoDataFrame, cars_per_person: float
+    ) -> int:
+        return 10
+
+    def _get_total_population(
+        population_data: gpd.GeoDataFrame, cars_per_person: float
+    ) -> int:
+        return 30
+
+    monkeypatch.setattr(route, "_get_num_people_on_route", _get_num_people_on_route)
+    monkeypatch.setattr(route, "_get_total_population", _get_total_population)
```

Comment:
Do we need to mock these functions? They don't send any requests out.

Reply:
I guess it's to avoid reading population data from files. You could make your own mock GeoDataFrame and feed it to both functions

Reply:
Also seems like this test will always pass because the mocked `_get_num_people_on_route` always returns 10